extensions/community_extensions/Protected Audience Support.md

```diff
@@ -106,6 +106,14 @@ Must include at least a buyer object (`igb`, in the bid response from the buyer
     <td>object</td>
     <td>The buyer’s priority signals, an object mapping string keys to Javascript numbers. If specified, the seller will add to its auction config <code>perBuyerPrioritySignals</code> attribute map, keyed by the Interest Group buyer origin. See https://github.com/WICG/turtledove/blob/main/FLEDGE.md#35-filtering-and-prioritizing-interest-groups </td>
   </tr>
+
+  <tr>
+    <td><code>begid</code></td>
+    <td>int</td>
+    <td>The buyer’s experiment group ID, an integer between zero and 65535 (16 bits). If specified, the seller will add to its auction config <code>perBuyerExperimentGroupIds</code> attribute map, keyed by the Interest Group buyer origin. See https://github.com/WICG/turtledove/blob/main/FLEDGE.md#21-initiating-an-on-device-auction 
+    <b>NOTE:</b> Assuming the auction is not run in parallel, the seller will provide the value via the <code>perBuyerExperimentGroupIds</code> auction configuration, provided the seller does not start the auction in parallel with OpenRTB requests.
```

Comment:
I find the references "not run in parallel" at the beginning and "not start the auction in parallel" at the end of the NOTE confusing and a little unclear; suggest instead:

> <b>NOTE:</b> The value of <code>perBuyerExperimentGroupIds</code> will only be available and provided when the seller does not start the auction in parallel with OpenRTB requests.